internal/packager/init.go

```diff
@@ -122,6 +122,45 @@ func checkComposeFileVersion(compose map[string]interface{}) error {
 	return schema.Validate(compose, fmt.Sprintf("%v", version))
 }
 
+func getEnvFiles(svcName string, envFileEntry interface{}) []string {
+	var envFiles []string
+	switch envFileEntry.(type) {
+	case string:
+		envFiles = append(envFiles, envFileEntry.(string))
+	case []string:
+		envFiles = append(envFiles, envFileEntry.([]string)...)
+	case []interface{}:
+		for _, env := range envFileEntry.([]interface{}) {
+			envFiles = append(envFiles, env.(string))
+		}
+	default:
+		logrus.Warnf("unknown entries in 'env_file' for service %s -> %v\n",
+			svcName, envFileEntry)
+		return envFiles
+	}
+	return envFiles
+}
+
+func checkEnvFiles(cfgMap map[string]interface{}) {
+	services, ok := cfgMap["services"]
+	if !ok {
+		return
+	}
+	servicesMap := services.(map[string]interface{})
+	for svcName, svc := range servicesMap {
```

Comment:
We unfortunately don't get a strongly typed model returned from compose yaml parsing, and use of `map[string]interface{}` make this bit fragile imho. Could you pleaes always check for successful type assertion?

Reply:
Done!
Thanks!